contracts/ERC721A.sol

```diff
@@ -364,8 +364,13 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
         // Clear approvals from the previous owner
         _approve(address(0), tokenId, prevOwnership.addr);
 
-        _addressData[from].balance -= 1;
-        _addressData[to].balance += 1;
+        // Underflow of the sender's balance is impossible because we check for
+        // ownership above and the recipient's balance can't realistically overflow.
```

Comment:
Lg- IMO worth noting this assumption at the top of file (that this assumes no owner can have more than uint128 of supply) even though it's extremely unlikely.

Reply:
Just added the comment!